packages/catalog-model/api-report.md

```diff
@@ -126,7 +126,7 @@ export const ENTITY_META_GENERATED_FIELDS: readonly ["uid", "etag", "generation"
 export type EntityEnvelope = {
     apiVersion: string;
     kind: string;
-    metadata: {
+    metadata: JsonObject & {
```

Comment:
Curious, given this, should the top-level object also have `JsonObject` mixed in, since we have additional values there too in practice?

Reply:
Ya quite right!

Reply:
fix'd